src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/UpToDate/BuildUpToDateCheck.Log.cs

```diff
@@ -252,7 +252,7 @@ public void UpToDate(int copyCount)
                     (TelemetryPropertyName.UpToDateCheck.AcceleratedCopyCount, copyCount)
                 });
 
-                Info(nameof(Resources.FUTD_UpToDate));
+                Info(nameof(VSResources.FUTD_UpToDate));
```

Comment:
You will also need to update `GetResourceString` on line 63 to refer to `VSResources` instead of `Resources` otherwise the  strings won't be found (GitHub won't let me leave a comment on that line 😕).